src/Components/Web/src/RendererRegistryEventDispatcher.cs

```diff
@@ -18,8 +18,7 @@ public static class RendererRegistryEventDispatcher
         /// For framework use only.
         /// </summary>
         [JSInvokable(nameof(DispatchEvent))]
-        public static Task DispatchEvent(
-            BrowserEventDescriptor eventDescriptor, string eventArgsJson)
+        public static Task DispatchEvent(BrowserEventDescriptor eventDescriptor, string eventArgsJson)
```

Comment:
This method could ultimately move to the Mono implementation as well, because we don't need it.

Reply:
That's sounds great, I can file a bug and we can clean it up. I'm not sure there might not be things lurking down here and I don't want to increase the scope of this PR even more.